### PR TITLE
Add filters UI to the Play List

### PR DESCRIPTION
**Pull request by AdriOli134**
Add the comboBox for select the file formats to filter them, and edit the appearance of the DatePicker

### CODE REVIEW

File: app/src/main/java/ju/video/player/view/commons/Fonts.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+/**
+ * It is for the stablish the constant fonts
+ *
+ * @author Adriana Olivera
+ * @version 1.0
+ */
+
+public class Fonts {
+    public static final Font TITLE_FONT = new Font("Impact", Font.BOLD, 25);
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/commons/Fonts.java]: The classes should be Singular. Change Fonts to Font.

File: app/src/main/java/ju/video/player/model/Formats.java [hunk 2]
```diff
@@ -8,25 +8,28 @@
...
 
+/**
+ * It is a enum of all the accepted formats with its respective mime type
+ *
+ * @author Sarai Alvarez
+ * @version 1.0
+ */
+
 public enum Formats {
+    VIDEO_MP4("video/mp4"),
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/model/Formats.java:19]: The classes should be Singular. Please change Formats to Format.

File: app/src/main/java/ju/video/player/view/materialDesing/constants/Constants.java [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+ * @author Libertad Tobalada
+ * @version 1.0
+ */
 
-	//public static final String RESOURCES_DIRECTORY = System.getProperty("user.dir")+"\\app\\src\\main\\java\\ju\\video\\player\\view\\materialDesing\\resources";
-	//public static final String RESOURCES_IMAGES = System.getProperty("user.dir")+"\\app\\src\\main\\resources";
-	public static final String RESOURCES_DIRECTORY = "app/src/main/java/ju/video/player/view/materialDesing/resources";
-	public static final String RESOURCES_IMAGES = "app/src/main/resources";
+public class Constants {
+	public static final String RESOURCES_DIRECTORY = "app/src/main/java/ju/video/player/view/materialDesing/resources/";
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/materialDesing/constants/Constants.java]: The classes should be Singular. Please set the class name in singular.

File: app/src/main/java/ju/video/player/view/playlist/filterspanel/DateComponentsPanel.java [hunk 4]
```diff
@@ -8,53 +8,47 @@
...
-        add(calendarIcon);
-        //calendarIcon.addMouseListener(new CalendarIconController(field));
-		
-        add(Box.createRigidArea(new Dimension(10, 0)));
-        
+        JLabel label = new JLabel(title);
+        label.setFont(Fonts.COMMON_FONT);
+        label.setForeground(Colors.COMPONETS_COLOR);
+		JTextField text = new JTextField(20);
+		Button buttonIcon = new Button("");
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/playlist/filterspanel/DateComponentsPanel.java:53]: Please align the code.

File: app/src/main/java/ju/video/player/view/playlist/filterspanel/DateComponentsPanel.java [hunk 5]
```diff
@@ -8,53 +8,47 @@
...
+        buttonIcon.setIcon("calendar-icon.png", 20, 20);
+		buttonIcon.addActionListener(new DatePickerController(this, text));
+        add(label, BorderLayout.WEST);
+		add(text, BorderLayout.CENTER);
+		add(buttonIcon, BorderLayout.EAST);
     }
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/playlist/filterspanel/DateComponentsPanel.java:59]: Please align the code.

File: app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java [hunk 6]
```diff
@@ -1,99 +1,144 @@
...
-			button[x].setFocusPainted(false);
-			button[x].setBackground(Color.white);
-			if (x > 6)
-				button[x].addActionListener(new ActionListener() {
+		for (int index = 0; index < button.length; index++) {
+			final int selection = index;
+			button[index] = new JButton();
+			button[index].setFocusPainted(false);
+			button[index].setForeground(Colors.COMPONETS_COLOR);
+			if (index > 6)
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java]: I am not sure what does means x > 6. Maibe 6 can set declare in a CONSTANT.

File: app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java [hunk 7]
```diff
@@ -1,99 +1,144 @@
...
 					public void actionPerformed(ActionEvent ae) {
 						day = button[selection].getActionCommand();
-						d.dispose();
+						dialog.dispose();
 					}
 				});
-			if (x < 7) {
-				button[x].setText(header[x]);
-				button[x].setForeground(Color.red);
+			if (index < 7) {
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java]: 7 should be a constant.

File: app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java [hunk 8]
```diff
@@ -1,99 +1,144 @@
...
+	/**
+	 * This update the calendar of the dialog
+	 */
 	public void displayDate() {
-		for (int x = 7; x < button.length; x++)
-			button[x].setText("");
-		java.text.SimpleDateFormat sdf = new java.text.SimpleDateFormat(
-				"MMMM yyyy");
-		java.util.Calendar cal = java.util.Calendar.getInstance();
+		for (int index = 7; index < button.length; index++) {
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java]: 7 should be a constant

**Review by MariaAT18 — APPROVED** [review]

File: app/src/main/java/ju/video/player/view/commons/ComboBox.java [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+ *
+ * @author Adriana Olivera
+ * @version 1.0
+ */
+
+public class ComboBox extends JComboBox<String> implements ItemListener {
+    
+    public ComboBox(String[] itemsList) {
+        addItems(itemsList);
+        setBounds(10,10,80,20);
```
Comment (junior0123) [inline app/src/main/java/ju/video/player/view/commons/ComboBox.java]: Perhaps you might consider saving those values in final variables if they are not going to change.
You are missing a space between the comma and the number.

File: app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java [hunk 10]
```diff
@@ -1,99 +1,145 @@
...
+ */
+
 public class DatePicker {
-	int month = java.util.Calendar.getInstance().get(java.util.Calendar.MONTH);
-	int year = java.util.Calendar.getInstance().get(java.util.Calendar.YEAR);;
-	JLabel l = new JLabel("", JLabel.CENTER);
-	String day = "";
-	JDialog d;
-	JButton[] button = new JButton[49];
+	private static final int WEEKDAYS = 7;
```
Comment (junior0123) [inline app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java:37]: Maybe it's just my impression, but I think it's tabbed more than the necessary 4 spaces. If it is a visualization problem of mine, please ignore this comment.

**Review by junior0123 — CHANGES_REQUESTED** [review]

File: app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java (hunk 10 above)
Comment (AdriOli134) [reply app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java:37]: please, in my computer I see all normal, I don't kwon if it is a problem of my configuration or of the gift, please

**Review by junior0123 — APPROVED** [review]

File: app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java (hunk 10 above)
Comment (junior0123) [reply app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java:37]: Must be a git bug

**Pull request merged**